src/components/actions/coreActions.ts

```diff
@@ -0,0 +1,484 @@
+      event_code: '',
+      sequence: -4,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: '=',
+            children: [
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.latestDecision.decision'],
+              },
+              'LIST_OF_QUESTIONS',
+            ],
+          },
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.isLastLevel'],
+          },
+        ],
+      },
+      parameter_queries: {},
+    },
+    // Will always increment stage if and only if last-level decision is
+    // "CONFORM". Any other cases for incrementing stage must be specified in
+    // template actions.
+    {
+      code: 'incrementStage',
+      path: '../plugins/action_increment_stage/src/index.ts',
+      name: 'Increment Stage',
+      trigger: 'ON_REVIEW_SUBMIT',
+      event_code: null,
+      sequence: -3,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.isLastLevel'],
+          },
+          {
+            operator: '=',
+            children: [
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.latestDecision.decision'],
+              },
+              'CONFORM',
+            ],
+          },
+        ],
+      },
+      parameter_queries: {},
+    },
+    // Change status to "CHANGES_REQUIRED" if last level and decision = LOQ
+    {
+      code: 'changeStatus',
+      path: '../plugins/action_change_status/src/index.ts',
+      name: 'Change Status',
+      trigger: 'ON_REVIEW_SUBMIT',
+      event_code: null,
+      sequence: -2,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: '=',
+            children: [
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.latestDecision.decision'],
+              },
+              'LIST_OF_QUESTIONS',
+            ],
+          },
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.isLastLevel'],
+          },
+        ],
+      },
+      parameter_queries: {
+        isReview: false,
+        newStatus: 'CHANGES_REQUIRED',
+      },
+    },
+    // Change outcome accordingly if final level and stage is CONFORM or
+    // NON_CONFORM
+    {
+      code: 'changeOutcome',
+      path: '../plugins/action_change_outcome/src/index.ts',
+      name: 'Change Outcome',
+      trigger: 'ON_REVIEW_SUBMIT',
+      event_code: null,
+      sequence: -1,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.latestDecision.decision'],
+          },
```

Comment:
Please read carefully the suggestion below...
I advise you had used the suggestion instead of re-doing it, https://github.com/openmsupply/conforma-server/commit/316aa22fe0251ea0d6a5586580b078008629f7e6 is still wrong.

Reply:
I actually did do it, but I committed to #978 instead, sorry.

Done here now: ecb08ebc